downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md

```diff
@@ -0,0 +1,337 @@
+  |--------------------------------------------------|----------------------------------------------------|
+  | `getCurrentRuntime()`                            | `io.ballerina.runtime.api.Runtime`                 |
+  | `createDecimalValue(String, DecimalValueKind)`   | `io.ballerina.runtime.api.creators.ValueCreator`   |
+  | `createStreamingJsonValue(JsonDataSource)`       | `io.ballerina.runtime.api.creators.ValueCreator`   |
+  | `convertToJson(Object, List<TypeValuePair>)`     | `io.ballerina.runtime.api.utils.JsonUtils`         |
+  | `getStringValue(Object, BLink)`                  | `io.ballerina.runtime.api.utils.StringUtils`       |
+  | `getExpressionStringValue(Object, BLink)`        | `io.ballerina.runtime.api.utils.StringUtils`       |
+  | `parseExpressionStringValue(String, BLink)`      | `io.ballerina.runtime.api.utils.StringUtils`       |
+  | `getValueKind()`                                 | `io.ballerina.runtime.api.values.BDecimal`         |
+  | `getStrand()`                                    | `io.ballerina.runtime.api.values.BFuture`          |
+  | `call(Strand, String, Object...)`                | `io.ballerina.runtime.api.values.BObject`          |
+  | `start(Strand, String, Object...)`               | `io.ballerina.runtime.api.values.BObject`          |
+  | `getRegExpDisjunction()`                         | `io.ballerina.runtime.api.values.BRegexpValue`     |
+  | `instantiate(Strand)`                            | `io.ballerina.runtime.api.values.BTypedesc`        |
+  | `instantiate(Strand, BInitialValueEntry[])`      | `io.ballerina.runtime.api.values.BTypedesc`        |
+
+### Bug fixes
+
+To view bug fixes, see the [GitHub milestone for 2201.6.0 (Swan Lake)](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A2201.6.0+label%3ATeam%2FjBallerina+label%3AType%2FBug+is%3Aclosed).
+
+## Standard library updates
+
+### New features
+
+#### `constraint` package
+
+- Introduced the `@constriant:Date` annotation to validate date record structures.
+- Allowed constraint annotations on subtypes.
+
+#### `http` package
+
+- Added constraint validation support for `query`, `path`, and `header` parameters.
+- Exposed the `http:Request` object in the response interceptors as a remote method parameter.
+- Added finite type support for `query`, `path`, and `header` parameters.
+
+#### `graphql` package
+
+- Added support for GraphQL field interceptors.
+- Added support for GraphQL interceptor configurations.
+- Added support to access subfields of a GraphQL field from the `graphql:Field` object.
+
+#### `persist` package
+
+- Added support for the `in-memory` data store.
+- Added support for the `google-sheets` data store. This is currently an experimental feature and its behavior may be subject to change in future releases.
+- Added support for defining enums and adding `enum` fields to the data model.
+
```

Comment:
#### `edi` package
- Introduced the `edi` module to convert EDI to JSON and JSON to EDI based on a given schema.

Reply:
Updated.